{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/utils/models.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import print_function
```

Comment:
unnecessary, Python 2 is not going to be used for any future projects (and there should never be any `print`s in production anyway)